Userland/Services/SystemServer/Service.cpp

```diff
@@ -198,13 +198,17 @@ ErrorOr<void> Service::spawn(int socket_fd)
             TRY(Core::System::setenv("SOCKET_TAKEOVER"sv, socket_takeover_builder.string_view(), true));
         }
 
-        if (m_account.has_value() && m_account.value().uid() != getuid()) {
-            auto& account = m_account.value();
-            if (auto error_or_void = account.login(); error_or_void.is_error()) {
-                dbgln("Failed to drop privileges (GID={}, UID={}), due to {}\n", account.gid(), account.uid(), error_or_void.error());
-                exit(1);
+        // NOTE: Dropping privileges only make sense when `SystemServer --user` is running
```

Comment:
Dropping privileges makes a lot of sense for `SystemServer` (without `--user`) - we already do this a lot.

Also typo: `make` -> `makes`